testbot.js

```diff
@@ -1,6 +1,14 @@
+// import all the requirements at the top for readability

```

Comment:
Move all the imports / requirements to the top
Makes the script much easier to read and generally helps with organisation